File: src/main/java/instance/common/LoadBlock.java
package instance.common;

import se.sics.kompics.Event;

import java.util.List;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-03-19
 *
 */

public class LoadBlock extends Event {

	private List<Block> blocks;

	public LoadBlock(List<Block> blocks) {
		this.blocks = blocks;
	}

	public List<Block> getBlocks() {
		return blocks;
	}

}

File: src/main/java/instance/common/LoadBlock.scala
package instance.common

import se.sics.kompics.Event
import java.util.{List => JList}

/**
 * @author Amir Moulavi
 * @date 2011-07-16
 *
 */
class LoadBlock(blocks:JList[Block]) extends Event {
  def getBlocks = blocks
}